src/frontend/.eslintrc.js

```diff
@@ -47,6 +47,7 @@ module.exports = {
     "sort-keys": "off",
     "prettier/prettier": 0,
     "@typescript-eslint/no-var-requires": 0,
+    "no-empty-pattern": 0, //there is a need in Playwright to pass empty { }
```

Comment:
Where is this used? I hesitate to add this because it will affect our entire codebase.

Reply:
`test.beforeAll(async ({}, workerInfo) => {`
we usually page, context in here but in this particular case there are not needed and throw lint error.
However, I have taken you feedback into account and refactored the tests so beforeAll is not needed (there are only two tests and the gain is a couple of seconds)